src/clr-angular/emphasis/alert/alert.ts

```diff
@@ -8,13 +8,20 @@ import { ChangeDetectorRef, Component, EventEmitter, Input, Optional, Output } f
 // providers
 import { AlertIconAndTypesService } from './providers/icon-and-types.service';
 import { MultiAlertService } from './providers/multi-alert.service';
+import { ClrCommonStrings } from '../../utils/i18n';
 
-@Component({ selector: 'clr-alert', providers: [AlertIconAndTypesService], templateUrl: './alert.html' })
+@Component({
+  selector: 'clr-alert',
+  providers: [AlertIconAndTypesService],
+  templateUrl: './alert.html',
+  styles: [':host { display: block; }'],
```

Comment:
What was this for?

Reply:
I wrote about it in the commit message and description of this PR:
>The CSS change on alerts is to fix a weird screenreader issue
due to the lack of display: block. No idea why.

We always want to do this anyway, we forget about it most the time these days but it leads to very strange styling behavior. The problem is that custom elements (our Angular component hosts) are always `display: inline` by default, when most of them contain block elements.

Reply:
👍